src/bpf/profiler.bpf.c

```diff
@@ -340,7 +340,7 @@ bpf_map_lookup_or_try_init(void *map, const void *key, const void *init) {
 static __always_inline void add_stack(struct bpf_perf_event_data *ctx,
                                       u64 pid_tgid,
                                       unwind_state_t *unwind_state) {
-  stack_count_key_t *stack_key = &unwind_state->stack_key;
+  stack_count_key_t *stack_key = &unwind_state->raw_stack.stack_key;
```

Comment:
nit: at this stage there's no raw stack and processed ones, it's just a stack :D 
```suggestion
  stack_count_key_t *stack_key = &unwind_state->stack.stack_key;
```

Reply:
Makes sense. Tried renaming this struct from `raw_stack_t` to `stack_t` and there was typedef redifinition error

```
./src/bpf/profiler.h:182:3: error: typedef redefinition with different types ('struct stack_t' vs 'struct sigaltstack')
         } stack_t;
           ^
         ./src/bpf/vmlinux_x86.h:17532:28: note: previous definition is here
         typedef struct sigaltstack stack_t;
```

Maybe we can rename the typedef to `stack_sample_t` (open to better naming suggestions)? Or do we keep the typedef as `raw_stack_t` and rename just the member variable in the unwind_state?

Reply:
Either works, don't want to block you here, we can always change it down the line!

Reply:
Thanks!